src/shared/config/externalUrls.ts

```diff
@@ -4,6 +4,7 @@ import { stringifyUrl } from '../utils/url';
 import { FileFormat } from '../types/resultsDownload';
 
 const IntActBase = '//www.ebi.ac.uk/intact/';
+const CommunityAnnotation = 'https://community.uniprot.org/bbsub';
```

Comment:
Changing the pathname here from `cgi-bin` to `bbsub` makes the logic checking the number of results fail, or in this case think that there are no annotation so display no link, because the custom response headers with the count are not returned.

It's a bit inconsistent from the server, because this one redirects to bbsub when following the link but would not work for `CommunityCurationAdd` 🤷🏽

Reply:
Reverted the currently used urls.